lib/std/http/Client.zig

```diff
@@ -488,7 +488,7 @@ pub const Response = struct {
             var line_it = mem.splitSequence(u8, line, ": ");
             const header_name = line_it.next().?;
             const header_value = line_it.rest();
-            if (header_value.len == 0) return error.HttpHeadersInvalid;
+            if (header_name.len == 0) return error.HttpHeadersInvalid;
```

Comment:
Previous code had this:
```
            var line_it = mem.tokenizeAny(u8, line, ": ");
            const header_name = line_it.next() orelse return error.HttpHeadersInvalid;
            const header_value = line_it.rest();
```

The `orelse return error.HttpHeadersInvalid` is dead code, so I'm not sure what that was supposed to do.

Looks like the name must be >= 1 byte [according to HTTP spec](https://www.rfc-editor.org/rfc/rfc7230#section-3.2.4) but value may be empty.

So I think this fix is correct.

Reply:
std/http/Server.zig needs the same fix.